src/main/java/de/tum/in/www1/artemis/domain/metis/Post.java

```diff
@@ -65,6 +66,11 @@ public class Post extends Posting {
     @Column(name = "display_priority")
     private DisplayPriority displayPriority;
 
+    @OneToOne
+    @JoinColumn(name = "plagiarism_case_id")
+    @JsonIncludeProperties({ "id" })
+    private PlagiarismCase plagiarismCase;
```

Comment:
Do we actually need this references? I would prefer a unidirectional reference from PlagiarismCase to Post